test/image/mocks/histogram2dcontour_bingroup-coloraxis.json

```diff
@@ -0,0 +1,756 @@
+{
+  "data": [
```

Comment:
This one looks like:

![image](https://user-images.githubusercontent.com/6675409/58580321-d2dd7480-821a-11e9-9687-2781b73deaf6.png)

on master.